lfucg-exactions/client/js/actions/apiActions.js

```diff
@@ -1348,7 +1348,7 @@ export function getPagination(page) {
     };
 }
 
-export function searchQuery() {
+export function searchQuery(csv) {
```

Comment:
For clarity, can we rename this parameter to isCsv or isCSV? (assuming it is just being used as a boolean indicator)

Reply:
It is being used that way and... done.